### PR TITLE
GH-563 LiteCommands 2.8.9 -> 3x.

### PR DESCRIPTION
**Pull request by Rollczi**
## Commands:
- [x] adminchat \<message>
- [x] afk 
- [x] alert \<type> \<message>
- [x] anvil 
- [x] anvil \<player>
- [x] automessage 
- [x] back 
- [x] back \<player>
- [x] butcher 
- [x] butcher \<chunks>
- [x] butcher \<chunks> \<mobType>
- [x] cartography \<player>
- [x] cartography 
- [x] chat clear
- [x] chat off
- [x] chat on
- [x] chat slowmode\<time>
- [x] clear \<player>
- [x] clear 
- [x] day 
- [x] day \<world>
- [x] delhome \<home>
- [x] delwarp\<warp>
- [x] disposal 
- [x] enchant \<enchantment> \<level> \<player>
- [x] enchant \<enchantment> \<level>
- [x] enderchest \<player>
- [x] enderchest 
- [x] eternalcore reload
- [x] feed 
- [x] feed \<player>
- [x] fly 
- [x] fly \<player>
- [x] gamemode \<gamemode> \<player>
- [x] gamemode \<gamemode>
- [x] gamemode 
- [x] give \<item> \<player>
- [x] give \<item>
- [x] give \<item> \<amount> \<player>
- [x] give \<item> \<amount>
- [x] god 
- [x] god \<player>
- [x] grindstone 
- [x] grindstone \<player>
- [x] hat 
- [x] heal 
- [x] heal \<player>
- [x] helpop \<message>
- [x] home 
- [x] home \<home>
- [x] ignore \<player>
- [x] ignore -all
- [x] invsee armor\<player>
- [x] invsee enderchest\<player>
- [x] invsee inventory\<player>
- [x] itemflag \<item-flag>
- [x] itemflag clear
- [x] itemlore \<line> \<text>
- [x] itemlore clear
- [x] itemname \<name>
- [x] itemname clear
- [x] kill 
- [x] kill \<player>
- [x] language 
- [x] list 
- [x] loom 
- [x] loom \<player>
- [x] msg\<player> \<message>
- [x] night 
- [x] night \<world>
- [x] online 
- [x] ping \<player>
- [x] ping 
- [x] rain \<world>
- [x] rain 
- [x] repair 
- [x] repair all
- [x] repair armor
- [x] reply\<message>
- [x] rtp \<player>
- [x] rtp 
- [x] sethome 
- [x] sethome \<home>
- [x] setspawn 
- [x] setwarp
- [x] skull \<player>
- [x] smithingtable \<player>
- [x] smithingtable 
- [x] socialspy
- [x] spawn \<player>
- [x] spawn 
- [x] speed \<speed>
- [x] speed \<speed> \<player>
- [x] speed \<type> \<speed>
- [x] speed \<type> \<speed> \<player>
- [x] stonecutter 
- [x] stonecutter \<player>
- [x] sun \<world>
- [x] sun 
- [x] teleport \<player> \<x> \<y> \<z> \<world>
- [x] teleport \<player>
- [x] teleport \<x> \<y> \<z> \<world>
- [x] teleport \<player> \<target-player>
- [x] teleportorandomplayer 
- [x] tellraw \<player> \<notice_type> \<message>
- [x] tellraw -all\<notice_type> \<message>
- [x] thunder 
- [x] thunder \<world>
- [x] time add\<time>
- [x] time add\<time> \<world>
- [x] time set\<time>
- [x] time set\<time> \<world>
- [x] tpa \<player>
- [x] tpaaccept \<player>
- [x] tpaaccept -all
- [x] tpadeny \<player>
- [x] tpadeny -all
- [x] tphere \<player>
- [x] tppos \<player> \<x> \<y> \<z>
- [x] tppos \<x> \<y> \<z>
- [x] tpup 
- [x] unignore \<player>
- [x] unignore -all
- [x] warp
- [x] warp\<warp>
- [x] whois \<player>
- [x] workbench \<player>
- [x] workbench

### CODE REVIEW

File: eternalcore-core/src/main/java/com/eternalcode/core/EternalCore.java [hunk 1]
```diff
@@ -14,7 +14,9 @@
 import com.eternalcode.core.publish.event.EternalShutdownEvent;
 import com.eternalcode.core.publish.Publisher;
 import net.dzikoysk.cdn.entity.Contextual;
+import net.dzikoysk.cdn.serdes.Serializer;
 import org.bukkit.Server;
```
Comment (P1otrulla) [inline eternalcore-core/src/main/java/com/eternalcode/core/EternalCore.java:18]: te importy po co?

**Review by P1otrulla — APPROVED** [review]
chyba gites

**Review by vLuckyyy — APPROVED** [review]

File: eternalcore-core/src/main/java/com/eternalcode/core/feature/chat/ChatManagerCommand.java [hunk 2]
```diff
@@ -76,24 +77,42 @@ void disable(Viewer viewer, CommandSender sender) {
             .send();
     }
 
-    @Execute(route = "slowmode", required = 1)
-    @DescriptionDocs(description = "Sets slowmode for chat", arguments = "<time>")
-    void slowmode(Viewer viewer, @Arg Duration duration) {
+    @Execute(name = "slowmode")
+    @DescriptionDocs(description = "Sets SlowMode for chat", arguments = "<time>")
```
Comment (Jakubk15) [inline eternalcore-core/src/main/java/com/eternalcode/core/feature/chat/ChatManagerCommand.java:81]: nie trzeba aż tak kapitalizować tego `SlowMode`, to nie jest jakoś specjalnie nazwa własna, ale jak chcesz
```suggestion
    @DescriptionDocs(description = "Sets slowmode for chat", arguments = "<time>")
```

**Review by imDMK — APPROVED** [review]

File: eternalcore-plugin/build.gradle.kts [hunk 3]
```diff
@@ -34,6 +34,6 @@ eternalShadowCompiler {
 
 tasks {
     runServer {
-        minecraftVersion("1.20")
+        minecraftVersion("1.20.2")
```
Comment (Jakubk15) [inline eternalcore-plugin/build.gradle.kts:37]:
```suggestion
        minecraftVersion("1.20.4")
```

**Review by Jakubk15 — APPROVED** [review]
Bardzo ładnie 🥱

File: eternalcore-plugin/build.gradle.kts (hunk 3 above)
Comment (vLuckyyy) [reply eternalcore-plugin/build.gradle.kts:37]: Paper 1.20.4 is experimental and unstable, currently I want to stay with 1.20.2 until the stable version is released.

**Pull request merged**